LICENSE.md

```diff
@@ -19,3 +19,6 @@ AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
 LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 SOFTWARE.
+
+The package contains public sector information licensed under the Open Government Licence v3.0. 
```

Comment:
I think the exact wording required by E is in Section 8.4 [here](https://digital.nhs.uk/about-nhs-digital/terms-and-conditions#8-your-use-of-nhs-england-content)
>“Contains information from NHS England, licenced under the current version of the Open Government Licence”.

Reply:
Also, as this is a markdown file you can/should probably link to the license https://www.nationalarchives.gov.uk/doc/open-government-licence/version/3